workflow/scripts/convert_fusions_to_vcf.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Does this script come from arriba sources? Please cite the full URL here.

Reply:
And is the license compatible?

Reply:
The license of arriba is also MIT and therefore compatible. :)